test/chef_authn_tests.erl

```diff
@@ -321,7 +487,7 @@ verify_sigs_v1_2_test_() ->
                                                       Sig,
                                                       list_to_binary(?X_OPS_USERID),
                                                       KeyList2,
-                                                      <<"1.2">>))
+                                                      {<<"sha1">>, <<"1.2">>}))
       end
     ].
 
```

Comment:
Can't comment where I want to; but authenticate_user_request_1_2_test_() @ line 536 might be worth extending to 1.3.